.github/workflows/ci.yml

```diff
@@ -13,10 +13,10 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
 
       - name: Setup node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3
```

Comment:
Hi @Ismailtlem and @GuillermoFloresV

This pinning suggestion from @renovate-bot is a terrible idea.

We should leverage GitHub, as a managed service, via `v3` instead of pinning and inheriting the maintenance responsibility.

Reply:
I am not very knowledgeable about that topic. Maybe @raisedadead can help or someone else from the core FCC team.

Reply:
This is a first because I have never seen an argument for not pinning dependencies 😅! 

I won't be able to advocate the rationale; please feel free to go ahead and research "Why you should ALWAYS pin dependencies". Or read up on the "12 factor app". 

I am happy to hear sensible arguments against it, though. Simply calling it a "terrible idea" lacks weight.

What's the maintenance overhead? 

It's pretty automated:

- Renovate sends PRs each weekend thanks to [our sane configs](https://github.com/freecodecamp/renovate-config). For security patches, it sends it immediately as soon as one is released.
- You write tests and employ CI to enforce those tests – which you will anyways for your code.
- Renovate is sending in dep updates and will only merge it (if auto-merge is enabled and applicable) with the CI passing.
- For JS dependencies, we auto-merge most updates; for GitHub workflows, we do not by design.

Your overhead is clicking the merge button when it passes!